### PR TITLE
Bug 1797486: Add icon next to pause/unpause events action

### PR DESCRIPTION
**Pull request by rawagner**
Screenshot from 2020-02-03 15-09-56

Screenshot from 2020-02-03 10-35-52

@andybraren

### CODE REVIEW

File: frontend/packages/console-shared/src/components/dashboard/activity-card/ActivityBody.tsx [hunk 1]
```diff
@@ -111,8 +112,18 @@ export const RecentEventsBody: React.FC<RecentEventsBodyProps> = (props) => {
     <>
       <div className="co-activity-card__recent-title">
         Recent Events
-        <DashboardCardButtonLink onClick={togglePause}>
-          {paused ? 'Unpause' : 'Pause'}
+        <DashboardCardButtonLink onClick={togglePause} className="co-activity-card__recent-actions">
+          {paused ? (
+            <>
+              <PlayIcon className="co-activity-item__icon" />
+              Unpause
```
Comment (spadgett) [inline frontend/packages/console-shared/src/components/dashboard/activity-card/ActivityBody.tsx]: Not new, but should this be "Resume"? I think would match the play icon better.

@andybraren ?

Comment (andybraren) [reply frontend/packages/console-shared/src/components/dashboard/activity-card/ActivityBody.tsx]: Agreed, was just thinking it felt a bit off with the new icon. 👍

Comment (rawagner) [reply frontend/packages/console-shared/src/components/dashboard/activity-card/ActivityBody.tsx]: changed to `Resume`

File: frontend/packages/console-shared/src/components/dashboard/activity-card/activity-card.scss [hunk 2]
```diff
@@ -55,6 +55,11 @@
   }
 }
 
+.co-activity-card__recent-actions.pf-c-button.pf-m-link.pf-m-inline {
```
Comment (spadgett) [inline frontend/packages/console-shared/src/components/dashboard/activity-card/activity-card.scss]: Does this style need to be so specific? I'd suggest

```suggestion
.co-activity-card__recent-actions {
```

or add another class with a BEM modifier if we need to restrict it to certain places.

Comment (rawagner) [reply frontend/packages/console-shared/src/components/dashboard/activity-card/activity-card.scss]: We dont need to restrict it, the style is used only in one place but PF4 is using styled components so If I just pass `co-activity-card__recent-actions` to PF4 button, my rules will still get overridden by theirs. Therefore I need to be more specific to get the priority in styling.

Comment (spadgett) [reply frontend/packages/console-shared/src/components/dashboard/activity-card/activity-card.scss]: I'd just use `!important` in that case. We're overriding something and we always want this rule to take precedence, so it makes sense here.

Comment (rawagner) [reply frontend/packages/console-shared/src/components/dashboard/activity-card/activity-card.scss]: Ok, if `!important` is fine I can actually go with a bit different approach and use `icon` prop of PF4 button. Currently there's a bug on PF side which causes the icon to be misaligned with I can fix that with `!important`

Can you take a look again ?

**Pull request merged**